App crashes, when set "Allow app to be used as launcher"

There is a problem, when set `Allow app to be used as launcher`. The app crashes immediately. It happens on a device with Android 9 (logs are attached) and also on another device with Android 11. Both devices are not rooted.

### Actual behaviour
The app crashes immediately, when switch on `Allow app to be used as launcher`.

### Expected behaviour
No crash and work😉
 
### Steps to reproduce
1. Go to `Settings` and enable `Allow app to be used as launcher`

### Can you reproduce the issue in demo mode?
Yes

### Environment data
#### Client
* Android version:  9 and 11
* App version: 2.20.13-beta
* Build flavor:  Full
* Device language: German

### Logs
#### App log
<details>
  <summary>Click to expand</summary>

```
--------- beginning of events
06-17 10:16:32.480 17539 17539 I am_on_paused_called: [0,org.openhab.habdroid.ui.LogActivity,performPause]
06-17 10:16:32.531 17539 17539 I am_on_restart_called: [0,org.openhab.habdroid.ui.PreferencesActivity,performRestartActivity]
06-17 10:16:32.552 17539 17539 I am_on_start_called: [0,org.openhab.habdroid.ui.PreferencesActivity,handleStartActivity]
06-17 10:16:32.554 17539 17539 I am_on_resume_called: [0,org.openhab.habdroid.ui.PreferencesActivity,RESUME_ACTIVITY]
--------- beginning of system
06-17 10:16:32.726 17539 17584 D HwCustConnectivityManagerImpl: isBlockNetworkRequestByNonAis, INVALID_SUBSCRIPTION_ID
06-17 10:16:32.875 17539 17539 I am_on_stop_called: [0,org.openhab.habdroid.ui.LogActivity,LIFECYCLER_STOP_ACTIVITY]
06-17 10:16:32.877 17539 17539 I am_on_destroy_called: [0,org.openhab.habdroid.ui.LogActivity,performDestroy]
06-17 10:16:32.877 17539 17539 D ActivityThread: Remove activity client record, r= ActivityRecord{8b7aeef token=android.os.BinderProxy@c73fa4e {org.openhab.habdroid.beta/org.openhab.habdroid.ui.LogActivity}} token= android.os.BinderProxy@c73fa4e
--------- beginning of crash
06-17 10:16:38.868 17539 17539 E AndroidRuntime: FATAL EXCEPTION: main
06-17 10:16:38.868 17539 17539 E AndroidRuntime: Process: org.openhab.habdroid.beta, PID: 17539
06-17 10:16:38.868 17539 17539 E AndroidRuntime: java.lang.IllegalArgumentException: Component class org.openhab.habdroid.beta.ui.LauncherActivityAlias does not exist in org.openhab.habdroid.beta
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.os.Parcel.createException(Parcel.java:1957)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.os.Parcel.readException(Parcel.java:1921)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.os.Parcel.readException(Parcel.java:1871)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.content.pm.IPackageManager$Stub$Proxy.setComponentEnabledSetting(IPackageManager.java:4900)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.app.ApplicationPackageManager.setComponentEnabledSetting(ApplicationPackageManager.java:2466)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at org.openhab.habdroid.ui.PreferencesActivity$MainSettingsFragment.onCreatePreferences$lambda-14(PreferencesActivity.kt:420)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at org.openhab.habdroid.ui.PreferencesActivity$MainSettingsFragment.$r8$lambda$nipPvD4U2LcfEn5gM2126OoqZBM(Unknown Source:0)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at org.openhab.habdroid.ui.PreferencesActivity$MainSettingsFragment$$ExternalSyntheticLambda17.onPreferenceChange(Unknown Source:0)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at androidx.preference.Preference.callChangeListener(Preference.java:1131)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at androidx.preference.TwoStatePreference.onClick(TwoStatePreference.java:71)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at androidx.preference.Preference.performClick(Preference.java:1198)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at androidx.preference.Preference.performClick(Preference.java:1182)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at androidx.preference.SwitchPreference.performClick(SwitchPreference.java:201)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at androidx.preference.Preference$1.onClick(Preference.java:182)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.view.View.performClick(View.java:6659)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.view.View.performClickInternal(View.java:6631)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.view.View.access$3100(View.java:790)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.view.View$PerformClick.run(View.java:26187)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.os.Handler.handleCallback(Handler.java:907)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.os.Handler.dispatchMessage(Handler.java:105)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.os.Looper.loop(Looper.java:216)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.app.ActivityThread.main(ActivityThread.java:7625)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at java.lang.reflect.Method.invoke(Native Method)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: Caused by: android.os.RemoteException: Remote stack trace:
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at com.android.server.pm.PackageManagerService.setEnabledSetting(PackageManagerService.java:23418)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at com.android.server.pm.PackageManagerService.setComponentEnabledSetting(PackageManagerService.java:23153)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at com.android.server.pm.HwPackageManagerService.setComponentEnabledSetting(HwPackageManagerService.java:1068)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at android.content.pm.IPackageManager$Stub.onTransact(IPackageManager.java:1550)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: at com.android.server.pm.PackageManagerService.onTransact(PackageManagerService.java:4473)
06-17 10:16:38.868 17539 17539 E AndroidRuntime: 
06-17 10:16:39.413 17697 17697 D ActivityThread: Attach thread to application
06-17 10:16:39.850 17697 17697 D HwCustConnectivityManagerImpl: isBlockNetworkRequestByNonAis, INVALID_SUBSCRIPTION_ID
```

</details>

--- a/mobile/src/main/AndroidManifest.xml
+++ b/mobile/src/main/AndroidManifest.xml
@@ -110,7 +110,7 @@
             </intent-filter>
         </activity>
         <activity-alias
-            android:name=".ui.LauncherActivityAlias"
+            android:name="${applicationId}.ui.LauncherActivityAlias"
             android:exported="true"
             android:enabled="false"
             android:targetActivity=".ui.MainActivity">

